internal/service/collection_handler.go

```diff
@@ -31,6 +31,9 @@ type ListRequest struct {
 
 	// ParentID is the identifier of the parent collection.
 	ParentID string
+
+	// Filter is the expression for filtering the results.
+	Filters []string
```

Comment:
The request already contains a `Selector` field with the result of parsing the `filters` parameter. Can you use that?